openpype/pipeline/frames.py

```diff
@@ -0,0 +1,62 @@
+"""Utility functions.
+
+This module contains utility functions dedicated to operations
+"""
+
+
+from openpype.lib import Logger
+
+log = Logger.get_logger(__name__)
+
+
+def get_frame_start_str(frame_start, frame_end):
+    """Get frame start string.
+
+    Args:
+        frame_start (int): first frame
+        frame_end (int): last frame
+
+    Returns:
+        str: frame start string
+    """
+    log.debug("frame_start, frame_end: {}, {}".format(frame_start, frame_end))
```

Comment:
I'm quite sure both these debug logs are overly verbose currently. Since all logging is handled in OpenPype I'd remove these.
```suggestion
```

Reply:
addressed here 488a2ff7a093f5c0aafe11deb09e5088dc7bfa64